src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -367,6 +367,14 @@ public static int startLibFuzzer(List<String> args) {
       // https://github.com/llvm/llvm-project/blob/da3623de2411dd931913eb510e94fe846c929c24/compiler-rt/lib/fuzzer/FuzzerFlags.def#L19
       args.add("-len_control=100");
     }
+
+    for (String arg : args.subList(1, args.size())) {
+      if (arg.length() > 0 && arg.charAt(0) != '-') {
```

Comment:
Assuming that empty args should never appear in practice, this could be easier to read:
```suggestion
      if (!arg.startsWith("-")) {
```